src/Plugins/ComplexCore/src/ComplexCore/Filters/ReplaceElementAttributesWithNeighborValuesFilter.cpp

```diff
@@ -0,0 +1,106 @@
+#include "ReplaceElementAttributesWithNeighborValuesFilter.hpp"
+
+#include "ComplexCore/Filters/Algorithms/ReplaceElementAttributesWithNeighborValues.hpp"
+
+#include "complex/DataStructure/DataPath.hpp"
+#include "complex/Filter/Actions/EmptyAction.hpp"
+#include "complex/Parameters/ArraySelectionParameter.hpp"
+#include "complex/Parameters/BoolParameter.hpp"
+#include "complex/Parameters/ChoicesParameter.hpp"
+#include "complex/Parameters/GeometrySelectionParameter.hpp"
+#include "complex/Parameters/NumberParameter.hpp"
+
+using namespace complex;
+
+namespace complex
+{
+//------------------------------------------------------------------------------
+std::string ReplaceElementAttributesWithNeighborValuesFilter::name() const
+{
+  return FilterTraits<ReplaceElementAttributesWithNeighborValuesFilter>::name.str();
+}
+
+//------------------------------------------------------------------------------
+std::string ReplaceElementAttributesWithNeighborValuesFilter::className() const
+{
+  return FilterTraits<ReplaceElementAttributesWithNeighborValuesFilter>::className;
+}
+
+//------------------------------------------------------------------------------
+Uuid ReplaceElementAttributesWithNeighborValuesFilter::uuid() const
+{
+  return FilterTraits<ReplaceElementAttributesWithNeighborValuesFilter>::uuid;
+}
+
+//------------------------------------------------------------------------------
+std::string ReplaceElementAttributesWithNeighborValuesFilter::humanName() const
+{
+  return "Replace Element Attributes with Neighbor (Threshold)";
+}
+
+//------------------------------------------------------------------------------
+std::vector<std::string> ReplaceElementAttributesWithNeighborValuesFilter::defaultTags() const
+{
+  return {"#Processing", "#Cleanup"};
+}
+
+//------------------------------------------------------------------------------
+Parameters ReplaceElementAttributesWithNeighborValuesFilter::parameters() const
+{
+  Parameters params;
+
+  params.insertSeparator(Parameters::Separator{"Input Parameters"});
+
+  params.insert(std::make_unique<Float32Parameter>(k_MinConfidence_Key, "Threshold Value", "", 0.1F));
+  params.insert(std::make_unique<ChoicesParameter>(k_SelectedComparison_Key, "Comparison Operator", "", 0, ChoicesParameter::Choices{"< [Less Than]", "> [Greater Than]"}));
```

Comment:
To meet PR checklist standards, choices should be placed as an enum in the header and dealt with as an enum. See WriteASCIIDataFilter for an example